exonum-java-binding/core/src/main/java/com/exonum/binding/core/service/Schema.java

```diff
@@ -27,8 +28,11 @@
  * the core automatically tracks every Merkelized collection used by the user
```

Comment:
```suggestion
 * Exonum automatically tracks every Merkelized collection used by the user
```
not sure that we use "core" term in public docs.